bitnami/mongodb/values.yaml

```diff
@@ -1104,3 +1148,311 @@ metrics:
+    ##
+    minAvailable: 1
+    ## Max number of pods that can be unavailable after the eviction
+    ##
+    # maxUnavailable: 1
+
+  ## Enable persistence using Persistent Volume Claims
+  ## ref: http://kubernetes.io/docs/user-guide/persistent-volumes/
+  ##
+  persistence:
+    enabled: true
+    ## A manually managed Persistent Volume and Claim
+    ## Requires persistence.enabled: true
+    ## If defined, PVC must be created manually before volume will be bound
+    ## Ignored when mongodb.architecture=replicaset
+    ##
+    # existingClaim:
+    ## PV Storage Class
+    ## If defined, storageClassName: <storageClass>
+    ## If set to "-", storageClassName: "", which disables dynamic provisioning
+    ## If undefined (the default) or set to null, no storageClassName spec is
+    ## set, choosing the default provisioner.
+    ##
+    # storageClass: "-"
+    ## PV Access Mode
+    ##
+    accessModes:
+      - ReadWriteOnce
+    ## PVC size
+    ##
+    size: 8Gi
+    ## PVC annotations
+    ##
+    annotations: {}
+    ## The path the volume will be mounted at, useful when using different
+    ## MongoDB(R) images.
+    ##
+    mountPath: /bitnami/mongodb
+    ## The subdirectory of the volume to mount to, useful in dev environments
+    ## and one PV for multiple services.
+    ##
+    subPath: ""
+    ## Fine tuning for volumeClaimTemplates
+    ##
+    volumeClaimTemplates:
+      ## A label query over volumes to consider for binding (e.g. when using local volumes)
+      ## See https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#labelselector-v1-meta for more details
+      ##
+      selector:
+
+  ## Service parameters
+  ##
+  service:
+    ## Service name
+    ##
+    nameOverride: ""
+    ## Service type
+    ##
+    type: ClusterIP
+    ## MongoDB(R) service port
+    ##
+    port: 27017
+    ## MongoDB(R) service port name
+    ##
+    portName: mongodb
+    ## Specify the nodePort value for the LoadBalancer and NodePort service types.
+    ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#type-nodeport
+    ##
+    nodePort: ""
+    ## MongoDB(R) service clusterIP IP
+    ##
+    # clusterIP: None
+    ## Specify the externalIP value ClusterIP service type.
+    ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#external-ips
+    ##
+    externalIPs: []
+    ## Specify the loadBalancerIP value for LoadBalancer service types.
+    ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer
+    ##
+    # loadBalancerIP:
+    ## Specify the loadBalancerSourceRanges value for LoadBalancer service types.
+    ## ref: https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/#restrict-access-for-loadbalancer-service
+    ##
+    loadBalancerSourceRanges: []
+    ## Provide any additional annotations which may be required. Evaluated as a template
+    ##
+    annotations: {}
```

Comment:
Why do we need this block? Correct me if I'm wrong but I though we were using a headless service to ensure every hidden node can be accessed using its own FQDN, and the `external.hidden.service` provide access to these nodes outside the cluster. Am I right?

Reply:
removed